packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -175,17 +197,23 @@ export default defineComponent({
     CreateSpace,
     FilesViewWrapper,
     NoContentMessage,
+    Pagination,
     ResourceTiles,
     ResourceTable,
     SideBar,
     SpaceContextActions
   },
   setup() {
     const store = useStore()
+    const router = useRouter()
+    const route = useRoute()
     const clientService = useClientService()
     const { selectedResourcesIds } = useSelectedResources({ store })
     const { can } = useAbility()
     const { current: currentLanguage, $gettext } = useGettext()
+    const filterTerm = ref('')
+    const markInstance = ref(undefined)
+    const tilesTableRef = ref(undefined)
```

Comment:
Do you need the ref here? I don't see you using it.

Reply:
which ref are you talking about? I use all 3

Reply:
I mean `tilesTableRef`. You set in in the template, but I don't see that you actually using it... or am I missing something? 🤔

Reply:
i use it in the template, but yeah you are right i switched to a selector